src/main/java/com/google/edith/GroceryDataReader.java

```diff
@@ -0,0 +1,90 @@
+package com.google.edith;
+
+import com.opencsv.CSVReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+/** Processes data file of product prices and sizes at different
+  * stores and returns the best value item. */
+public class GroceryDataReader {
+    
+  /** Finds the specified product in the file and puts the
+    * data into DealItem objects to be handled. */
+  public DealItem readFile(String itemName) throws IOException {
+    URL csvresource = getClass().getClassLoader().getResource("grocerydata.csv");
+    File groceryDataFile = new File(csvresource.getFile());
+
+    CSVReader reader = new CSVReader(new FileReader(groceryDataFile), ',');
+    DealItem bestItem = null;
+
+    String[] record = null;
+    record = reader.readNext();
+    record = reader.readNext();
+
+	while ((record = reader.readNext()) != null) {
+      if (record[0].equals(itemName)) {
+        List<DealItem> dealItems = new ArrayList<DealItem>();
+
+	    DealItem item1 = new DealItem();
+        item1.setStore("Aldi");
+        item1.setPrice(record[1]);
+        item1.setWeight(record[2]);
+        item1.setComment(record[3]);
+        dealItems.add(item1);
+
+        DealItem item2 = new DealItem();
+        item2.setStore("Kroger");
```

Comment:
iiuc these store names are pervasive, so we should be able to move them to private static final vars at the top of the class. 
If there are other such common properties that are used in other classes then we can move them to a static utility class as well and make them public static final vars. Lmk if you have any questions 

`private static final String KROGER = "Kroger";`